### PR TITLE
[WIP][ADD] New module 'stock_reception_screen'

### PR DESCRIPTION
**Pull request by sebalix**
Dedicated screen to scan and receive goods.

Development status: alpha

### CODE REVIEW

**Comment by pedrobaeza** [comment]
@sebalix did you have the chance to check `stock_barcodes` module?

**Comment by sebalix** [comment]
@pedrobaeza not yet (or not myself), I just move the PR for now.

File: stock_reception_screen/models/stock_reception_screen.py [hunk 1]
```diff
@@ -0,0 +1,455 @@
...
+        validate it.
+        It is performed right after the processing of the current move (so
+        before checking the next move to process).
+        """
+        if self.current_move_line_id and self.current_move_id:
+            remaining_qty = (
+                self.current_move_id.product_uom_qty
+                - self.current_move_line_id.qty_done)
+            new_move_id = self.current_move_id._split(remaining_qty)
+            new_move = self.env["stock.move"].browse(new_move_id)
```
Comment (i-vyshnevska) [inline stock_reception_screen/models/stock_reception_screen.py]: this line looks redundant

Comment (sebalix) [reply stock_reception_screen/models/stock_reception_screen.py]: indeed, unused variable. fixed

**Comment by sebalix** [comment]
It was decided internally to move this PR to https://github.com/OCA/wms/pull/10 for dependency reasons, closing.

**Pull request closed without merging**